modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-deploy.adoc

```diff
@@ -86,11 +102,17 @@ spec:
   clusterSpec: {}
 ----
 +
+- `metadata.name`: Name to assign the Redpanda cluster. This name is also assigned to the Helm release.
+- xref:reference:crd.adoc#chartref[`spec.chartRef`]: Information about the Helm chart that will be used to deploy Redpanda.
+- xref:reference:crd.adoc#redpandaclusterspec[`spec.clusterSpec`]: This is where you can override default values in the Redpanda Helm chart.
```

Comment:
Same here, the ID I see in the .adoc file is `{anchor_prefix}-github-com-redpanda-data-redpanda-src-go-k8s-apis-redpanda-v1alpha1-redpandaclusterspec`